server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
If the figure is monthly contributions off a fixed budget, we can go with `/flattr/contributions/`.